cvcl/forms.py

```diff
@@ -1,6 +1,12 @@
 from django.forms import ModelForm, ModelChoiceField
 from django import forms
-from .models import Assignment, VmDefinition
+from .models import Assignment, VmDefinition, User
+import io
+import csv
+
+
+# from django.contrib.auth.models import User
```

Comment:
I will remove the comment. I realized we didn't need it earlier when our User model inherits from django.contrib.auth.models.User